subsystememu/src/Main.scala

```diff
@@ -41,6 +43,7 @@ object Main {
         case a => Some(a)
       }
     os.write(dir_ / s"$topName.anno.json", firrtl.annotations.JsonProtocol.serialize(annos))
+    freechips.rocketchip.util.ElaborationArtefacts.files.foreach { case (ext, contents) => os.write.over(dir_ / s"${p.toString}.${ext}", contents()) }
```

Comment:
do we need this?

Reply:
Yes. We need some `.tcl` scripts generated by using ElaborationArtefacts.